homeassistant/components/device_tracker/bluetooth_tracker.py

```diff
@@ -17,24 +17,31 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-REQUIREMENTS = ['pybluez==0.22']
+REQUIREMENTS = ['pybluez==0.22', 'bt_proximity==0.1.2']
 
 BT_PREFIX = 'BT_'
 
+CONF_REQUEST_RSSI = 'request_rssi'
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Optional(CONF_TRACK_NEW): cv.boolean
+    vol.Optional(CONF_TRACK_NEW): cv.boolean,
+    vol.Optional(CONF_REQUEST_RSSI): cv.boolean
 })
 
 
 def setup_scanner(hass, config, see, discovery_info=None):
     """Set up the Bluetooth Scanner."""
     # pylint: disable=import-error
     import bluetooth
+    from bt_proximity import BluetoothRSSI
 
-    def see_device(device):
+    def see_device(mac, name, rssi=None):
         """Mark a device as seen."""
-        see(mac=BT_PREFIX + device[0], host_name=device[1],
-            source_type=SOURCE_TYPE_BLUETOOTH)
+        attributes = {}
+        if rssi is not None:
+            attributes['rssi'] = rssi
+        see(mac=BT_PREFIX + mac, host_name=name,
```

Comment:
Better use `mac="%_%s" % (BT_PREFIX, mac)` I suppose :-)

Reply:
There you go! 😉